app/views/practices/show/authenticated/_show_authenticated.html.erb

```diff
@@ -48,17 +24,36 @@
         </li>
       <% end %>
     </ul>
-    <div class="text-center">
-      <%= link_to 'See adoption checklist', practice_planning_checklist_url(practice_id: @practice.slug), id: 'next-steps-link-in-nav', class: 'usa-button width-full p1' %>
+    <%
+      main_email = @practice.support_network_email
+      practice_emails = @practice.practice_emails
+    %>
+    <div class="text-center margin-bottom-1">
+      <%= mail_to main_email, 'Email practice', cc: practice_emails.map(&:address).join(', '), class: 'usa-button--outline dm-btn-primary width-full display-flex flex-align-center flex-justify-center height-5', 'aria-label': "side nav email #{main_email}"  %>
+    </div>
+    <div class="text-center margin-bottom-1">
+      <%= link_to 'Start adoption', practice_planning_checklist_url(practice_id: @practice.slug), id: 'next-steps-link-in-nav', class: 'usa-button dm-btn-primary width-full' %>
     </div>
+    <% if @practice.user_id == current_user.id || current_user.roles.any? %>
+      <div class="text-center">
+        <%= link_to practice_instructions_path(@practice), id: 'edit-link-in-nav', class: 'usa-button dm-btn-base width-full' do %>
+          <i class="fas fa-edit margin-right-1"></i>Edit practice
+        <% end %>
+      </div>
+    <% end %>
   </div>
 </div>
+
 <div class="practice-mobile-nav padding-1 z-top desktop:display-none display-flex justify-center">
-  <%= link_to 'See adoption checklist', practice_planning_checklist_url(practice_id: @practice.slug), id: 'next-steps-link-in-mobile-nav', class: 'usa-button' %>
+  <%= mail_to main_email, 'Email', cc: practice_emails.map(&:address).join(', '), id: 'email-link-in-mobile-nav', class: 'width-100 usa-button--outline dm-btn-primary display-flex flex-align-center flex-justify-center height-5 x0-875-right', 'aria-label': "Mobile nav email #{main_email}"  %>
```

Comment:
how many times is this `practice_emails.map(&:address).join(', ')` used on the page?
can we make this a variable to use in those spots?

Reply:
I created a variable for this one as well as a couple of others. Thank you for catching that!